buildkite-agent/plan.ps1

```diff
@@ -0,0 +1,24 @@
+$pkg_name="buildkite-agent"
+$pkg_origin="core"
+$pkg_version="3.9.1"
+$pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+$pkg_license=@("MIT")
+$pkg_description="The Buildkite Agent is an open-source toolkit written in Golang for securely running build jobs on any device or network."
+$pkg_build_deps=@("core/7zip")
+$pkg_source="https://github.com/buildkite/agent/releases/download/v${pkg_version}/buildkite-agent-windows-amd64-${pkg_version}.zip"
+$pkg_filename="buildkite-agent-windows-amd64-${pkg_version}.zip"
+$pkg_shasum="b46203b515288982b476898f24904244063b7869be4df494d63d7243d915773c"
+$pkg_upstream_url="https://buildkite.com"
+$pkg_bin_dirs=@("bin")
+
+function Invoke-Unpack {
```

Comment:
In windows plans you can drop the 7zip dep and this function. The default `Invoke-Unpack` will expand a source zip file into `$HAB_CACHE_SRC_PATH/$pkg_dirname`.